src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -565,14 +564,14 @@ const WorkflowView = _.flow(
           div({ style: { marginTop: '0.5rem' } }, [
             'Snapshot ',
             sourceRepo === 'agora' ?
-              div({ style: { display: 'inline-block', marginLeft: '0.25rem', minWidth: 60 } }, [
+              div({ style: { display: 'inline-block', marginLeft: '0.25rem', minWidth: 75 } }, [
                 h(Select, {
                   isDisabled: !!Utils.editWorkspaceError(ws),
                   isClearable: false,
                   isSearchable: false,
                   value: methodVersion,
                   getOptionLabel: ({ value }) => Utils.normalizeLabel(value),
-                  options: _.uniq([...snapshotIds, savedConfig.methodRepoMethod.methodVersion]).sort(),
+                  options: _.sortBy(_.toNumber, _.uniq([...snapshotIds, savedConfig.methodRepoMethod.methodVersion])),
```

Comment:
So the list goes `1, 2, 11` instead of `1, 11, 2`

Reply:
Are these always numbers?

Reply:
Apparently in agora they're always int's.